### PR TITLE
Search - show content type, author and content

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
### Todo: 

- [x] Add content type to search.json
- [x] Show content type
- [x] Add author to search.json if author
- [x] Show author 
- [x] Add content to search.json
- [x] Show summary of content

### Screenshot

zoeken defacto

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -152,15 +152,28 @@
...
       to_store[:title] = resource.data.title.send(I18n.locale)
     end
+    if resource.url.start_with?("/blog/")
+      to_store[:type] = "Blog"
+    elsif resource.url.start_with?("/jobs/")
+      to_store[:type] = "Vacature" if root_locale == :nl
+      to_store[:type] = "Vakanz" if root_locale == :de
+    else
+      to_store[:type] = "Pagina" if root_locale == :nl
+      to_store[:type] = "Seite" if root_locale == :de
```
Comment (fatboypunk) [inline config.rb]: Can we I18n this?

Comment (RonaldDijkstra) [inline config.rb]: Yes we can

File: source/javascripts/search.js [hunk 2]
```diff
@@ -52,13 +52,26 @@
...
+            html += '<span class="separator">–</span>' + page.author;
+          }
+
+          html += '</span><p>';
+
+          var content = page.content;
+          var shortText = jQuery.trim(content).substring(0, 400)
+                          .split(' ').slice(0, -1).join(' ') + '...';
+
+          html += shortText +
```
Comment (sn3p) [inline source/javascripts/search.js]: Could simplify this without having to create an additional variable:

```js
html += $.trim(content).substring(0, 400)
        .split(' ').slice(0, -1).join(' ') + '...';
```

File: source/javascripts/search.js [hunk 3]
```diff
@@ -52,13 +52,26 @@
...
+
+          '&nbsp;<a href="' + page.url + '">Lees verder</a></p>' +
```
Comment (sn3p) [inline source/javascripts/search.js]: Need to i18n this I guess?
We're [already passing](https://github.com/DefactoSoftware/website/blob/master/source/localizable/zoeken.html.erb#L14) translations via data-attrs.

Comment (RonaldDijkstra) [inline source/javascripts/search.js]: Thanks

**Comment by sn3p** [comment]
💯

**Pull request merged**